utils/resultstable.go

```diff
@@ -339,11 +339,13 @@ func PrepareLicenses(licenses []services.License) ([]formats.LicenseRow, error)
 
 // Prepare secrets for all non-table formats (without style or emoji)
 func PrepareSecrets(secrets []*sarif.Run) []formats.SourceCodeRow {
-	return prepareSecrets(secrets, false)
+	preparedSecrets, _ := prepareSecrets(secrets, false)
```

Comment:
Used for SimpleJson, make sure the values are also added to this format or you need to pass the bool

Reply:
simplejson works well